src/components/Post/GovernanceSideBar/Referenda/VoteReferendum.tsx

```diff
@@ -246,23 +287,81 @@ const VoteReferendum = ({ className, referendumId, onAccountChange, lastVote, se
 		let voteTx = null;
 
 		if(proposalType === ProposalType.OPEN_GOV){
-			voteTx = api.tx.convictionVoting.vote(referendumId, { Standard: { balance: lockedBalance, vote: { aye, conviction } } });
+
+			if(vote === EVoteDecisionType.AYE ) {
+				try {
```

Comment:
don't need this try catch here

Reply:
assigning to a variable won't throw any error

Reply:
Will remove it